src/interfaces/assistants_web/src/app/tasks/[agentId]/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
+
+import { useAgent, useAgentTasks } from '@/hooks/use-agents';
+
+type Props = {
+  params: {
+    agentId: string;
+  };
+  searchParams: Record<string, string>;
+};
+
+const Page = ({ params }: Props) => {
+  const agentId = params.agentId;
+  const {
+    data: agentTasks,
+    isLoading: isTaskLoading,
+    error: isTaskError,
+  } = useAgentTasks({ agentId });
+  const { data: agent, isLoading, error } = useAgent({ agentId });
```

Comment:
this component can be changed to RSC and either hydrate these queries (see other pages for example). Or directly call the agentTasks and agent endpoint and pass the values to the children component